app/webpacker/css/admin_v3/components/pagination.scss

```diff
@@ -34,8 +35,8 @@
       margin-left: 28px; // 28+8 (the gap) = 36px : the space between the arrows and the numbers
     }
 
-    &.disabled {
-      cursor: default;
+    &:not(.disabled):not(.current) {
+      cursor: pointer;
```

Comment:
I'm thinking we shouldn't need to specify the cursor at all.
If it has a `href`, it is clickable and the browser will decide to show the pointer. 
If it is disabled, then it should have no `href`. I see that currently, these `.disabled` elements have `href="#"`, but can this be removed?

Reply:
> If it has a href, it is clickable and the browser will decide to show the pointer.

Yes, but...
For the `current` element, it has a `text` cursor, so I need to set it as `default`. And then, I should set the not current|disabled to pointer. Reading this I could do also the exact opposite: setting to default only for current|disabled and let the pointer to be set as it's a link. Which is maybe easier to understand. 

Commit cfe79b8b79